products/mssqlserver.md

```diff
@@ -8,6 +8,12 @@ activeSupportColumn: true
 versionCommand: select @@version
 releaseDateColumn: true
 releases:
+-   releaseCycle: "2022"
+    releaseLabel: "2022"
```

Comment:
could you add `releaseLabel: "__RELEASE_CYCLE__"` to root so all cycles which doesnt have special
label will be their releaseCycle itself [ that will make your releaseLabel assignment not needed ]

Reply:
`releaseLabel` will have to be updated on the next SQL Server 2022 cumulative updates, so I think there is no need to remove it now.

@usta, `releaseLabel: "__RELEASE_CYCLE__"` is already the default value used in the product template : https://github.com/endoflife-date/endoflife.date/blob/a604acc9601c3f020e1ed89e518c257c7700a158/_layouts/product.html#L58